app/views/report_mailer/tables_report.html.erb

```diff
@@ -0,0 +1,85 @@
+<%#
+- @tables: an array of tables, expects first "row" to be an options hash
+%>
+
+<style type="text/css">
+
+.table-number {
+  font-variant-numeric: tabular-nums;
+  text-align: right;
+}
+
+<%#
+borrowed from markdown-here styling
+https://github.com/adam-p/markdown-here/blob/1df5aee09cae90e2991db4e9cfb8c1b65a3c1898/src/common/default.css
+%>
+table {
+  padding: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
+  font-size: 1em;
+  font: inherit;
+  border: 0;
+}
+
+tbody {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+table tr {
+  border: 0;
+  border-top: 1px solid #CCC;
+  background-color: white;
+  margin: 0;
+  padding: 0;
+}
+
+table tr:nth-child(2n) {
+  background-color: #F8F8F8;
+}
+
+table tr th, table tr td {
+  font-size: 1em;
+  border: 1px solid #CCC;
+  margin: 0;
+  padding: 0.5em 1em;
+}
+
+table tr th {
+ font-weight: bold;
+  background-color: #F0F0F0;
+}
+</style>
+
+<% @tables.each do |table| %>
+  <% options, header, *rows = table %>
+
+  <h2><%= options[:title] %></h2>
+
+  <table>
+    <thead>
+      <tr>
+        <% header.each do |head| %>
+          <th><%= head %></th>
+        <% end %>
+      </tr>
+    </thead>
+    <tbody>
+      <% rows.each do |row| %>
+        <tr>
+          <% row.each do |cell| %>
+            <td <%= cell.is_a?(Numeric) ? 'class=table-number' : nil %> >
+              <% if cell.is_a?(Float) && options[:float_as_percent] %>
```

Comment:
So, we'll either only be able to do all percents or all floats in each table... right?  I'm okay with that, we'll just build a separate report, if we need something more nuanced. 👍🏽

Reply:
Yeah that was my guess!